src/Microsoft.DotNet.CodeAnalysis/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "Microsoft.Net.Compilers.NetCore": "1.2.0-beta1-20160202-02",
+    "Microsoft.NETCore.Console": "1.0.0-rc2-23923",
+    "System.Reflection.Metadata": "1.3.0-rc2-23923",
+    "System.Runtime.Extensions": "4.1.0-rc2-23923",
+    "Microsoft.Composition": "1.0.30",
```

Comment:
Is this a new dependency that we will need to add to the tool-runtime project.json? Or are we already including it? I kind of remember ApiCompat using it.